### PR TITLE
Finish migration to prometheus-community

### PR DESCRIPTION
**Pull request by lilic**
I was not sure what to do with the Dockerfile, we can sort that in a follow up PR, as currently the go.mod file is not allowing the correct new import path.

cc @brancz

### CODE REVIEW

**Comment by brancz** [comment]
Thanks!

File: go.mod [hunk 1]
```diff
@@ -1,10 +1,11 @@
-module github.com/openshift/prom-label-proxy
+module github.com/prometheus-community/prom-label-proxy
 
 go 1.13
 
 require (
 	github.com/go-openapi/runtime v0.19.15
 	github.com/go-openapi/strfmt v0.19.5
+	github.com/openshift/prom-label-proxy v0.1.0 // indirect
```
Comment (simonpasquier) [inline go.mod:8]: there are probably .go files still importing from this.

Comment (lilic) [reply go.mod]: No think it was my local import that was wrong, should be fixed now, thanks!

Comment (lilic) [reply go.mod]: PTAL :)

**Pull request merged**